ShopParser/SenukaiParser.cs

```diff
@@ -0,0 +1,56 @@
+﻿using ItemLibrary;
+using Microsoft.VisualBasic;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+
+namespace ShopParser
+{
+    class SenukaiParser : IParser<Computer>
+    {
+        private string _url = "https://www.senukai.lt/c/kompiuterine-technika-biuro-prekes/nesiojami-kompiuteriai-ir-priedai/nesiojami-kompiuteriai/5ei";
+        private IWebDriver _driver;
+
+
+        public SenukaiParser ()
+        {
+            var options = new ChromeOptions();
+            options.AddArguments("--headless");
+            _driver = new ChromeDriver();
+        }
+
+
+        public static void Main (string[] args)
+        {
+            var test = new SenukaiParser();
+            var tet = test.ParseShop();
+        }
```

Comment:
Don't think we need a main here, we will be running from WPF, this should be classified as a class library

Reply:
removed Main